extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -871,6 +863,61 @@ export const AzureProvisionDialog: React.FC = () => {
                 onBack();
               }}
             />
+            <PrimaryButton
+              disabled={!formData.creationType}
+              style={{ margin: '0 4px' }}
+              text={formatMessage('Next')}
+              onClick={() => {
+                switch (formData.creationType) {
+                  case 'import':
+                    setPage(PageTypes.EditJson);
+                    setTitle(DialogTitle.EDIT);
+                    break;
+                  case 'create':
+                  case 'generate':
+                  default:
+                    setPage(PageTypes.ConfigProvision);
+                    setTitle(DialogTitle.CONFIG_RESOURCES);
+                    break;
+                }
+              }}
+            />
+          </div>
+        </div>
+      );
+    } else if (page === PageTypes.ConfigProvision) {
+      return (
+        <div style={{ display: 'flex', flexFlow: 'row nowrap', justifyContent: 'space-between' }}>
+          {currentUser ? (
+            <Persona
+              secondaryText={formatMessage('Sign out')}
+              size={PersonaSize.size40}
+              text={currentUser.name}
+              onRenderSecondaryText={onRenderSecondaryText}
+            />
+          ) : (
+            <div
+              style={{ color: 'blue', cursor: 'pointer' }}
+              onClick={() => {
+                clearAll();
+                closeDialog();
+                logOut();
+              }}
+            >
+              Sign out
+            </div>
+          )}
+          <div>
+            <DefaultButton
+              style={{ margin: '0 4px' }}
+              text={formatMessage('Back')}
+              onClick={() => {
+                clearAll();
+                setItem(profileName, formData);
+                setPage(PageTypes.ChooseAction);
```

Comment:
I highly recommend putting page and title into the same type and set them at once, since they always get set together.

Reply:
That would be a good idea for when this module gets completely rewritten. :)